### PR TITLE
fix: fix normalization total for table

### PR DESCRIPTION
**Pull request by jjxie**

### CODE REVIEW

**Comment by qlikossbuild** [comment]
Hi, and thanks for contributing!

Note: If you are part of Qlik R&D, you can apply to the `open-source` team in our _internal_ GitHub organization.

Before we can handle your pull request we need you to sign our contributor license agreement.
This is a one-time process and enables you to contribute without hassle to any of the qlik-oss repositories in the future.

And it only takes a few minutes! Here are some useful links to get you started:

* [Sign the CLA](https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sign-the-cla)
* [Rationale behind the CLA](https://github.com/qlik-oss/open-source/blob/master/cla-rationale.md)
* [Qlik R&D Open Source repository](https://github.com/qlik-oss/open-source#open-source-at-qlik-rd)
* [Official Slack team](http://branch.qlik.com/slack)

Having troubles? Feel free to reach out to us either here or on Slack!

**Note:** Once the CLA is sorted, you should rebase/touch this pull request to trigger an update of this CLA status.

File: src/index.js [hunk 1]
```diff
@@ -273,6 +301,31 @@ function limitedSorting({ measures, properties, layout }) {
   return hasActive && !needDims;
 }
 
+/**
+ * Check if one type of modifier use inter record function
+ * @param {String} modifierType
+ */
+function getIfUseInterRecordFunction(modifierType) {
+  let ifUseInterRecord = false;
```
Comment (quanho) [inline src/index.js]: availableModifiers[modifierType].useInterRecordFunction()

File: src/expression-modifiers/accumulation/index.js [hunk 2]
```diff
@@ -91,5 +91,7 @@ export default {
     helper.initModifier(modifier, DEFAULT_OPTIONS);
   },
 
+  useInterRecordFunction: () => true,
```
Comment (quanho) [inline src/expression-modifiers/accumulation/index.js]: enableTotalsFunction () => false,

File: src/expression-modifiers/normalization/index.js [hunk 3]
```diff
@@ -113,6 +113,8 @@ export default {
     }
   },
 
+  useInterRecordFunction: () => false,
```
Comment (quanho) [inline src/expression-modifiers/normalization/index.js]: enableTotalsFunction: (measure) => {
//do
},

**Pull request merged**